<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="2" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-03-20T20:01:36 IST" name="Default suite" finished-at="2023-03-20T20:01:36 IST" duration-ms="161">
    <groups>
    </groups>
    <test started-at="2023-03-20T20:01:36 IST" name="Default test" finished-at="2023-03-20T20:01:36 IST" duration-ms="161">
      <class name="TsetNgBasics.DependOnDemo">
        <test-method signature="login()[pri:0, instance:TsetNgBasics.DependOnDemo@4f6ee6e4]" started-at="2023-03-20T20:01:36 IST" name="login" finished-at="2023-03-20T20:01:36 IST" duration-ms="21" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="forgotPassword()[pri:0, instance:TsetNgBasics.DependOnDemo@4f6ee6e4]" started-at="2023-03-20T20:01:36 IST" name="forgotPassword" finished-at="2023-03-20T20:01:36 IST" duration-ms="4" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- forgotPassword -->
        <test-method signature="sendFriendRequest()[pri:0, instance:TsetNgBasics.DependOnDemo@4f6ee6e4]" started-at="2023-03-20T20:01:36 IST" name="sendFriendRequest" finished-at="2023-03-20T20:01:36 IST" depends-on-methods="TsetNgBasics.DependOnDemo.login" duration-ms="7" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Login is not Working]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Login is not Working
at org.testng.Assert.fail(Assert.java:110)
at TsetNgBasics.DependOnDemo.sendFriendRequest(DependOnDemo.java:18)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sendFriendRequest -->
        <test-method signature="acceptFriendRequest()[pri:0, instance:TsetNgBasics.DependOnDemo@4f6ee6e4]" started-at="2023-03-20T20:01:36 IST" name="acceptFriendRequest" finished-at="2023-03-20T20:01:36 IST" depends-on-methods="TsetNgBasics.DependOnDemo.sendFriendRequest, TsetNgBasics.DependOnDemo.login" duration-ms="1" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TsetNgBasics.DependOnDemo.acceptFriendRequest() on instance TsetNgBasics.DependOnDemo@4f6ee6e4 depends on not successfully finished methods 
[TsetNgBasics.DependOnDemo.sendFriendRequest() on instance TsetNgBasics.DependOnDemo@4f6ee6e4
TsetNgBasics.DependOnDemo.login() on instance TsetNgBasics.DependOnDemo@4f6ee6e4]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TsetNgBasics.DependOnDemo.acceptFriendRequest() on instance TsetNgBasics.DependOnDemo@4f6ee6e4 depends on not successfully finished methods 
[TsetNgBasics.DependOnDemo.sendFriendRequest() on instance TsetNgBasics.DependOnDemo@4f6ee6e4
TsetNgBasics.DependOnDemo.login() on instance TsetNgBasics.DependOnDemo@4f6ee6e4]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- acceptFriendRequest -->
      </class> <!-- TsetNgBasics.DependOnDemo -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
